docs/userguide/pyproject_config.rst

```diff
@@ -7,8 +7,8 @@ Configuring setuptools using ``pyproject.toml`` files
 .. note:: New in 61.0.0
 
 .. important::
-   For the time being, ``pip`` still might require a ``setup.py`` file
-   to support :doc:`editable installs <pip:cli/pip_install>`.
+   For the time being [#pep660-status]_, ``pip`` still might require a ``setup.py`` file
+   to support :doc:`editable installs <pip:cli/pip_install>` [#setupcfg-caveats]_.
```

Comment:
It does not -- and this note can be removed.

`pip` will gracefully figure out whether to use PEP 660, or invoke setup.py manually. I believe this was from a pre-PEP 660 implementation days.